### DISCUSSION TITLE
Checkbox wrapped by Controller generate by map doesn't work!

### DISCUSSION THREAD

**Discussion by sonphung205** [react-hook-form/react-hook-form#6224] [Q&A]
This is my example code https://codesandbox.io/s/beautiful-wood-ouy8x?file=/src/App.js

Steps
1. Check at least one of list chekcbox 1
2. Check at least one of list chekcbox 2
3. Submit and console show the correct values for list ids1, but list ids2 is not

Sorry for my bad english! I am so new for this community!

**Comment by bluebill1049** [comment]
`Controller` wrapped a group of checkboxes, not the other way around.

**Reply by sonphung205** [reply]
I just updated my sandbox but it's seem doesn't work, can you pls check it again?
